great_expectations/validation_operators/validation_operators.py

```diff
@@ -19,6 +19,42 @@
 # NOTE: Abe 2019/08/24 : This is first implementation of all these classes. Consider them UNSTABLE for now.
 
 
+class ValidationOperatorResult(object):
+    # TODO this is a design sketch
+    def __init__(self):
+        raise NotImplementedError
+
+    def get_run_id(self):
+        raise NotImplementedError
+
+    def list_assets_validated(self) -> list:
```

Comment:
Would the returned list look something like this:
I think it would be good to have some

[
    {
      "batch_kwargs": {
        "path": "/Users/me/projects/my_project/data/data.csv",
        "datasource": "my_filesystem_datasource",
        "reader_method": "read_csv"
      },
      "expectation_suite_names": ["suite_one", "suite_two"],
  "validation_result" : ...
    },
    {
      "batch_kwargs": {
        "query": "SELECT * FROM users WHERE status = 1",
        "datasource": "my_redshift_datasource"
      },
      "expectation_suite_names": ["suite_three"],
  "validation_result" : ...

    }
  ]


If yes, it would mirror the config object, with the addition of "validation_result" key to every asset.

Reply:
Yes this would work well for using in checkpoint